game/source_engine_nav.ksy

```diff
@@ -0,0 +1,468 @@
+# Base NAV file format in kaitai struct.
+meta:
+  id: source_engine_nav
+  title: NAV
+  file-extension: nav
+#  tags:
+#    - nav_mesh
+#    - source_engine
+#    - valve
+  license: CC0-1.0
+  endian: le
+
+doc: |
+  Base file format used by the Source Engine to store nav mesh data for maps. This KSY file does *not* account for derivative implementations.
+
+  Code for the base NAV file format can be found in `game/server/nav_file.cpp`, which is located in the Source SDK 2013 repository (located at `https://github.com/ValveSoftware/source-sdk-2013/`).
+
+doc-ref: "https://developer.valvesoftware.com/wiki/NAV"
+
+seq:
+  - id: magic_number
+    contents: [0xce, 0xfa, 0xed, 0xfe] # 0xFEEDFACE
+    doc: Magic Number of the NAV file. NAV files are always generated with 0xFEEDFACE in little-endian.
+  - id: version
+    type: u4
+    doc: |
+      Version of the base NAV file format. It is present in all NAV files.
+
+      Changelog over NAV versions:
```

Comment:
I don't think it is needed. But I have an idea.

Reply:
https://github.com/kaitai-io/kaitai_struct/issues/918

Reply:
It is mostly for convenience, though I'm thinking of moving the changelog to the global `doc`.